Tests/Simulation/IntegrationTestFixtureSim.cs

```diff
@@ -53,7 +53,7 @@ public override void SetupTestDevices()
             Device1002_Simulated_OTAA = new TestDeviceInfo()
             {
                 DeviceID = "0000000000001002",
-                AppEUI = "0000000000001002",
+                AppEUI = GetJoinEui(1002),
```

Comment:
`0000000000001002` was in hex so shouldn't this read `GetJoinEui(0x1002)`?

Reply:
Yes, it is not the same value before/after. I tried to be consistent with some of the other helper methods, where in the past we went away from passing hex numbers to just passing integers in decimal representation. Based on our discussion offline, we should probably revisit how we create these devices entirely.

Reply:
Update: This comment caught a valid bug. I thought that devices were recreated on E2E test runs if the properties do not match (for which we have logic, which is apparently not executed on each E2E run). I needed to revert to `JoinEui.Parse("...0027")` for the E2E tests to make sure that the EUIs still match the values of the twin.